ui/apps/platform/src/Components/CompoundSearchFilter/attributes/alert.ts

```diff
@@ -0,0 +1,18 @@
+// If you're adding a new attribute, make sure to add it to "alertAttributes" as well
+
+import { CompoundSearchFilterAttribute } from '../types';
+
+export const InactiveDeployment: CompoundSearchFilterAttribute = {
+    displayName: 'Inactive deployment',
+    filterChipLabel: 'Inactive deployment',
+    searchTerm: 'Inactive Deployment',
+    inputType: 'select',
+    inputProps: {
+        options: [
+            { value: 'true', label: 'True' },
+            { value: 'false', label: 'False' },
+        ],
+    },
+};
```

Comment:
Maybe a question for UX, but I think this would read more clearly if the third dropdown had a display name of "Deployment status" (or similar), and the values in the dropdown were "Inactive" and "Active" to represent true/false. Similarly for the filter chips.

Reply:
That's a good suggestion. I also thought it looked a bit odd, but didn't give it more thought. Now that I look at it again, it would definitely be an improvement.

 I wonder if it's worthwhile to keep this in the Advanced Filters vs. outside. The only thing is that it's specific to deployments. I feel like segmenting the violations into Workloads, Container Images, and Resources would be helpful so filters specific to certain types can show up only for those types. But since we're not doing that, maybe it's fine to keep it in the Advanced Filters